site/content/3.10/get-started/set-up-a-cloud-instance.md

```diff
@@ -138,10 +138,11 @@ Alternatively, follow the steps of the linked guides:
 
 ## Free-to-Try vs. Paid
 
-The ArangoGraph Insights Platform comes with a free-to-try tier that lets you test
-the ArangoDB Cloud for free for 14 days. It includes one project and one small
-deployment of 4GB, local backups, and one notebook for learning and data science.
-After the trial period, your deployment is automatically deleted.
+The ArangoGraph Insights Platform comes with a [perpetual free tier](../arangograph/organizations/_index.md#perpetual-free-trial)
+that lets you test the platform indefinitely at no cost. It includes one project
+and one small deployment of 4GB, local backups, and one notebook for learning
```

Comment:
```suggestion
and one small deployment of 4 GB, local backups, as well as one notebook for learning
```
Mention that it is 4 GB _of memory_ (per node)?

Maybe clarify what notebook means (don't make it sound like you get a free laptop 🙈 ), and what you can learn with it?